allianceutils/checks.py

```diff
@@ -268,13 +283,35 @@ def _check_explicit_table_names_on_model(model: Type[Model], enforce_lowercase:
     """
     errors = []
     found = None
-    tree = ast.parse(inspect.getsource(model))
+    try:
+        source = inspect.getsource(model)
+    except OSError:
+        # if a model class is dynamically created then we can't inspect the source code
+        # (eg this happens with audit models)
+        message = f"Can't get source for {model.__name__}"
+        # warnings.warn(message)
+        errors.append(Info(
+            message,
+            hint="Dynamically generated model source code can't be introspected",
+            obj=model,
+            id=ID_INFO_EXPLICIT_TABLE_NAME_LOWERCASE,
+        ))
+        #print(f"Cant get source for {model.__name__}")
+        return errors
```

Comment:
huh, why's this an error not just a warn?

Reply:
django system checks can be level info/warn/error

Was matching the pattern [in the docs](https://docs.djangoproject.com/en/dev/topics/checks/), but those were all errors. Have changed to `messages` (and added missing type info)

Also removed that print